website/static/events/london-summit-2024/index.html

```diff
@@ -0,0 +1,528 @@
+                                xmlns:xlink="http://www.w3.org/1999/xlink">
+                                <g id="Page-1" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
+                                    <g id="location_on_FILL1_wght400_GRAD0_opsz48" fill="#000000" fill-rule="nonzero">
+                                        <path
+                                            d="M12,14.625 C12.725,14.625 13.34375,14.36875 13.85625,13.85625 C14.36875,13.34375 14.625,12.725 14.625,12 C14.625,11.275 14.36875,10.65625 13.85625,10.14375 C13.34375,9.63125 12.725,9.375 12,9.375 C11.275,9.375 10.65625,9.63125 10.14375,10.14375 C9.63125,10.65625 9.375,11.275 9.375,12 C9.375,12.725 9.63125,13.34375 10.14375,13.85625 C10.65625,14.36875 11.275,14.625 12,14.625 Z M12,30 C7.975,26.575 4.96875,23.39375 2.98125,20.45625 C0.99375,17.51875 0,14.8 0,12.3 C0,8.55 1.20625,5.5625 3.61875,3.3375 C6.03125,1.1125 8.825,0 12,0 C15.175,0 17.96875,1.1125 20.38125,3.3375 C22.79375,5.5625 24,8.55 24,12.3 C24,14.8 23.00625,17.51875 21.01875,20.45625 C19.03125,23.39375 16.025,26.575 12,30 Z"
+                                            id="Shape"></path>
+                                    </g>
+                                </g>
+                            </svg>
+                            Microsoft Reactor<br />
+                            2 Kingdom Street<br />
+                            London W2 6BD
+                        </p>
+                    </div>
+                    <div class="grid-cell grid-cell--col6 grid-cell--col10--tablet">
+                        <p class="f32 f24--tablet paragraph-2">
+                            Come together with the
+                            <a href="https://vis.gl/" target="_blank" rel="noopener noreferrer">vis.gl</a>/<a
+                                href="https://deck.gl/" target="_blank" rel="noopener noreferrer">deck.gl</a>
+                            community to discuss the present & future of the leading open source
+                            library for geospatial applications.
+                        </p>
+                    </div>
+                </div>
+        
+                <div class="grid-cell grid-cell--col10 u-mt80 gradient-border u-flex-column u-align-center">
+                    <p class="f32 f24--tablet paragraph-3">
+                        At the Open Visualization Collaborator Summit, we'll be bringing
+                        together an international audience of geospatial minds to discuss how
+                        they are using deck.gl to build apps, foster more contribution and
+                        envisage the future of the leading open source mapping library.
+                    </p>
+                    <a href="linuxfoundation.regfox.com/open-visualization-collaborator-summit-2024" class="u-mt24 u-mb80 u-mb48--tablet button-gl">GET YOUR TICKET NOW</a>
```

Comment:
```suggestion
                    <a href="https://linuxfoundation.regfox.com/open-visualization-collaborator-summit-2024" class="u-mt24 u-mb80 u-mb48--tablet button-gl">GET YOUR TICKET NOW</a>
```

@Pessimistress link to register is broken on deployed site